src/app/manuals/PlayerPanelDemo.ts

```diff
@@ -0,0 +1,19 @@
+import { Player } from '../../Player';
+import { PlayerPanel } from '../components/PlayerPanel';
+import { Render } from '../utils/Render';
+import { View } from '../View';
+
```

Comment:
Consider adding a short line that says what should be inserted in app.ts so the view of the panel can be seen. Ex. :
`// PlayerPanelDemo.playDemo();`

Reply:
Added in description (just now). In future I will try to remember adding this :)